joomla/joomla-cms
# Add a paramter to com_contact to allow the mailto link to be switched on / off

#### Summary of Changes

This PR allows to set the mailto link for each contact individually.

With the additional change to move the cloak handling into the layout, it there can easily be overridden. Also, it is now possible to access the unmodified email address.

**Note: language files were changed and help screens might need to be updated.**
#### Test Instructions
1. **Back-end:** Create a contact. Give it a title and make sure to fill out the email address field. The contact doesn't have to be assigned to a user or a category. For easy access, create a menu item to that contact in a menu of your choice.
2. **Front-end:** Verify that the email address including the mailto link is visible in the front-end. If not, please check the settings for com_contact.
3. **Back-end:** Go to the component settings and set "Add Mailto: Link" to "No"
4. **Front-end:** Refresh the page. Without the patch you will see that the email address will always have a mailto link attached to it.
5. **Front-end** / **Back-end:** Apply the patch and check again. The mailto link can now be switched on or off via component, contact or menu item settings.
